src/main/java/de/chojo/jdautil/interactions/slash/Argument.java

```diff
@@ -72,16 +80,22 @@ public static ArgumentBuilder channel(String name, String description) {
         return builder(OptionType.CHANNEL, name, description);
     }
 
-    public static ArgumentBuilder number(String name, String description) {
+    public static NumberArgumentBuilder number(String name, String description) {
         return builder(OptionType.NUMBER, name, description);
     }
 
     public static ArgumentBuilder mentionable(String name, String description) {
         return builder(OptionType.MENTIONABLE, name, description);
     }
 
+    private OptionData applySize(OptionData data){
+        min().accept(data);
+        max().accept(data);
```

Comment:
This feels weird but given the implementation of the ArgumentBuilder, I can understand why you did it like this.

Reply:
It was really the only good way to combine the String with number limits without adding additional bloat